pkg/kubelet/kubelet.go

```diff
@@ -1350,6 +1350,9 @@ func (kl *Kubelet) GeneratePodHostNameAndDomain(pod *api.Pod) (string, string, e
 // GenerateRunContainerOptions generates the RunContainerOptions, which can be used by
 // the container runtime to set parameters for launching a container.
 func (kl *Kubelet) GenerateRunContainerOptions(pod *api.Pod, container *api.Container, podIP string) (*kubecontainer.RunContainerOptions, error) {
+	if pod.Status.PodIP == "" {
```

Comment:
Yeah, that's the main thing that is holding me back when trying to access pod IP in ConfigMap volume plugin.